<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;
color: white;</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>Doctor Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>Designation:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>License</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>Phone Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="doctorname">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color: black;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="designation">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color: black;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="license">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color: black;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_number">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color: black;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="address">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>340</y>
     <width>321</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color: black;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="doctorsubmitbutton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>530</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 248, 248);
color: black;
font-size: 30pt;
font: 20pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gobackdoctor">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>530</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 248, 248);
color: black;
font-size: 30pt;
font: 20pt &quot;Haettenschweiler&quot;;</string>
   </property>
   <property name="text">
    <string>Go Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
